src/server/scripts/Commands/cs_modify.cpp

```diff
@@ -80,23 +80,32 @@ class modify_commandscript : public CommandScript
         return commandTable;
     }
 
-    //Edit Player HP
-    static bool HandleModifyHPCommand(ChatHandler* handler, const char* args)
+    template<typename... Args>
+    static void NotifyModification(ChatHandler* handler, Unit* target, const TrinityStrings resourceMessage, const TrinityStrings resourceReportMessage, Args&&... args)
```

Comment:
Const non reference qualified parameters are converted into it's decayed version (the constness is dropped internally), which makes it unneccesarry to add const to the params here.